examples/fundamentals__typescript/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
do we need tsconfig.json for out of the box TS support though?

Reply:
Without it, `npm run tsc` doesn't work. I left comment on the json file.